contracts/rootchain/RootChain.sol

```diff
@@ -203,14 +208,14 @@ contract RootChain is Ownable {
         childBlock storage blk = childChain[txPos[0]];
 
         // check signatures
-        bytes32 merkleHash = keccak256(txBytes);
-        require(txHash.checkSigs(keccak256(abi.encodePacked(merkleHash, blk.root)), // confirmation hash -- sha3(merkleHash, root)
+        bytes32 merkleHash = sha256(txBytes);
+        require(txHash.checkSigs(sha256(abi.encodePacked(merkleHash, blk.root)), // confirmation hash -- sha256(merkleHash, root)
```

Comment:
the variable `txHash` is returned by the `decodeTransaction` helper, which creates `txHash` with a keccak hash: `txHash = keccak256(spendMsg[0].toRlpBytes());` (RootChain.sol line 182). Should that be changed to a sha256 hash as well?